src/components/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -255,10 +267,37 @@ export default {
 				this.mgAmount = autoDepositAmount - this.donation;
 				this.paymentMethod = data?.my?.autoDeposit?.paymentMethod ?? {};
 				this.subscriptionId = data?.my?.autoDeposit?.id;
+				this.firstName = data?.my?.userAccount?.firstName ?? '';
+				this.subscriptionsLoans = data?.mySubscriptions?.values?.length ?? 0;
+				const mySubscriptions = data?.mySubscriptions?.values?.history?.values ?? [];
+				this.subStartTimestamp = mySubscriptions?.[mySubscriptions.length - 1]?.timeStamp ?? null;
 			}
 		},
 	},
 	computed: {
+		subStartDate() {
```

Comment:
We use parseISO everywhere else in the ui app to convert UTC dates to formatted date strings. I'm not sure if there is a difference here, just wanted to point that out

Reply:
The `LenderSubscription.timestamp` field is a `Long` integer timestamp. It probably should have been setup initially as a `Date` UTC string, but that's not what we're working with, unfortunately.